src/components/docs-version-switcher/docs-version-switcher.test.tsx

```diff
@@ -208,3 +208,113 @@ describe('setProjectForAriaLabel', () => {
 		expect(result).toBe(`Terraform Intro`)
 	})
 })
+
+describe('TFE Releases - DocsVersionSwitcher', () => {
+	let options: VersionSelectItem[]
+	let wrapper: React.FC
+
+	beforeEach(() => {
+		options = [
+			{
+				name: 'latest',
+				label: 'v202401-1 (latest)',
+				isLatest: true,
+				releaseStage: 'stable',
+				version: 'v202401-1',
+			},
+			{
+				name: 'v202312-1',
+				label: 'v202312-1',
+				isLatest: false,
+				releaseStage: 'stable',
+				version: 'v202312-1',
+			},
+			{
+				name: 'v202311-1',
+				label: 'v202311-1',
+				isLatest: false,
+				releaseStage: 'stable',
+				version: 'v202311-1',
+			},
+			{
+				name: '',
+				label: 'v202310-1',
+				isLatest: false,
+				releaseStage: 'stable',
+				version: 'v202310-1',
+			},
+		]
+
+		wrapper = function Wrapper({ children }: { children: ReactNode }) {
+			return (
+				<CurrentProductProvider
+					currentProduct={
+						{
+							name: 'Terraform',
+							slug: 'terraform',
+							currentRootDocsPath: {
+								iconName: 'org',
+								name: 'Terraform Enterprise',
+								path: 'enterprise',
+								includeMDXSource: true,
+							},
+						} as $TSFixMe
```

Comment:
Do we know why we need to cast this type? Curious if there are plans to clear these up eventually